### PR TITLE
Migrate some FileUploader and FileDownloader APIs to User

### PR DESCRIPTION
**Pull request by ezaquarii**
Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

- [x] Tests written, or not not needed

### CODE REVIEW

**Comment by ezaquarii** [comment]
@AlvaroBrey 
`testServerChangedFileContent` timeout - probably not me.

File: app/src/main/java/com/owncloud/android/files/services/FileUploader.java [hunk 1]
```diff
@@ -600,9 +607,7 @@ public void uploadFile(String uploadKey) {
 
         if (mCurrentUpload != null) {
             /// Check account existence
-            if (!accountManager.exists(mCurrentUpload.getAccount())) {
-                Log_OC.w(TAG, "Account " + mCurrentUpload.getAccount().name +
-                    " does not exist anymore -> cancelling all its uploads");
```
Comment (AlvaroBrey) [inline app/src/main/java/com/owncloud/android/files/services/FileUploader.java:605]: Please restore this deleted log call

**Review by AlvaroBrey — CHANGES_REQUESTED** [review]
Thanks @ezaquarii, looks good except a deleted log call. Can you restore it, or explain your rationale for deleting it?

File: app/src/main/java/com/owncloud/android/files/services/FileUploader.java (hunk 1 above)
Comment (ezaquarii) [reply app/src/main/java/com/owncloud/android/files/services/FileUploader.java]: Done

**Comment by AlvaroBrey** [comment]
Thanks @ezaquarii !

**Pull request merged**